expenses/models.py

```diff
@@ -15,6 +15,7 @@ class Category(models.TextChoices):
         CULTURE = 'Culture'
         VACATIONS = 'Vacations'
         OTHER = 'Other'
+    description = models.TextField(max_length=250, default='')
```

Comment:
What is the puprpose of this field?
Adding extra text to the expense so the person can know what, where, on which ocasion etc.. ?
Just making sure